src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -118,538 +127,644 @@ export class JailAddComponent implements OnInit {
-      tooltip: T('List any commands to run in the system environment\
- before a jail is started.'),
+      placeholder: T('exec.prestart'),
+      tooltip: T('Commands to run in the system environment before a \
+                 before a jail is created.'),
     },
     {
       type: 'input',
       name: 'exec_poststart',
-      placeholder: T('exec_poststart'),
-      tooltip: T('List any commands to run in the system environment after\
- a jail is started and after any <b>exec_start</b> commands are finished.'),
-    }, {
+      placeholder: T('exec.poststart'),
+      tooltip: T('Commands to run in the system environment after a \
+                 jail is created and after any <b>exec.start</b> \
+                 commands are finished.'),
+    },
+    {
       type: 'input',
       name: 'exec_prestop',
-      placeholder: T('exec_prestop'),
-      tooltip: T('List any commands to run in the system environment\
- before a jail is stopped.'),
-    }, {
+      placeholder: T('exec.prestop'),
+      tooltip: T('Commands to run in the system environment before a \
+                 jail is removed.'),
+    },
+    {
       type: 'input',
       name: 'exec_poststop',
-      placeholder: T('exec_poststop'),
-      tooltip: T('List any commands to run in the system environment after\
- a jail is stopped.'),
-    }, {
+      placeholder: T('exec.poststop'),
+      tooltip: T('Commands to run in the system environment after a jail\
+                 is removed.'),
+    },
+    {
       type: 'checkbox',
       name: 'exec_clean',
-      placeholder: T('exec_clean'),
+      placeholder: T('exec.clean'),
       tooltip: T('Run commands in a clean environment. The current\
- environment is discarded except for HOME, SHELL, TERM and USER. HOME\
- and SHELL are set to the target login default values. USER is set to\
- the target login. TERM is imported from the current environment. The\
- environment variables from the login class capability database for the\
- target login are also set.'),
-    }, {
+                 environment is discarded except for $HOME, $SHELL, \
+                 $TERM, and $USER. <br>\
+                 $HOME and $SHELL are set to the target login. $USER is \
+                 set to the target login. $TERM is imported from the \
+                 current environment. The environment variables from the \
+                 login class capability database for the target login \
+                 are also set.'),
+    },
+    {
       type: 'input',
       name: 'exec_timeout',
-      placeholder: T('exec_timeout'),
-      tooltip: T('Define the maximum amount of time in seconds to wait for\
- a command to complete. If a command is still running after the allotted\
- time, the jail will be terminated.'),
-    }, {
+      placeholder: T('exec.timeout'),
+      tooltip: T('Maximum amount of time in seconds to wait for a \
+                  command to complete. If a command is still running \
```

Comment:
Sentence rework: "If a command is still running after the allotted time, the jail will not be created or removed." -> A jail cannot be created or removed when a command is still running after the allotted time.

Reply:
fixed